src/shared/components/ErrorAlert/reducers/errorAlert.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  ErrorAlertAction,
+  ErrorAlertAllActions,
+  ErrorAlertState,
+} from '../types/errorAlertTypes';
+
+const errorAlertState: ErrorAlertState = {
+  message: 'Something goes wrong',
```

Comment:
why do you have message in initial state

Reply:
fixed , I forgot to clear it